cla_backend/apps/cla_eventlog/models.py

```diff
@@ -48,6 +48,10 @@ def __unicode__(self):
 
     def save(self, *args, **kwargs):
         super(Log, self).save(*args, **kwargs)
+        if self.type == LOG_TYPES.OUTCOME:
+            logger.info('LGA-293 Saved outcome code {} (Log id: {}, Case ref:{})'.
+                        format(self.case.outcome_code, self.id, self.case.reference))
```

Comment:
🙋‍♂️ Would there be _any_ potential occurence of `self.case` being `None` here?

Reply:
It would raise an AttributeError, but `self.case` cannot be None here after the save because it's a required field on the model.